src/main/java/seedu/address/model/FilteredStudents.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model;
+
+import java.util.function.Predicate;
+
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import seedu.address.model.student.Student;
+
+/**
+ * Represents the students and type of information to be displayed on the GUI.
+ */
+public class FilteredStudents {
+    private final FilteredList<Student> filteredStudents;
+    // Only the student's name and ID should be displayed if true; all information should be shown otherwise
+    private boolean conciseInfo;
```

Comment:
```suggestion
    private boolean isConciseInfo;
```
Perhaps can change this to isConciseInfo to represent the boolean!

Reply:
Thanks for spotting the error. Will change it to hasConciseInfo instead as it is more grammatically correct.